File: 981-delete-columns-to-make-sorted/delete-columns-to-make-sorted.java
class Solution {
    public int minDeletionSize(String[] strs) {
        int c=0;
        for(int i=0;i<strs[0].length();i++){
            for(int j=0;j<strs.length-1;j++){
                if((strs[j].charAt(i)-0)>(strs[j+1].charAt(i)-0)){
                    c++;
                    break;
                }
            }
        }
        return c;
    }
}

File: 981-delete-columns-to-make-sorted/delete-columns-to-make-sorted.js
/**
 * @param {string[]} strs
 * @return {number}
 */
var minDeletionSize = function(strs) {
    let c=0;
    for(var i=0;i<strs[0].length;i++){
        for(var j=0;j<strs.length-1;j++){
            if(strs[j][i]>strs[j+1][i]){
                c++;
                break;
            }
        }
    }
    return c;
};